static/docker-compose.yml

```diff
@@ -91,7 +127,7 @@ volumes:
   dbdata: {}
   uploads: {}
   letsencrypt: {}
-  redis:
+  redis: 
```

Comment:
👀

Reply:
:thonk:

Reply:
this is how I pad my github stats.

Reply:
in my defense, this friggin yaml file doesn't adhere to any known convention so I can't use autoformatting